gdcdictionary/schemas/core_metadata_collection.yaml

```diff
@@ -36,72 +36,72 @@ required:
   - submitter_id
   - type
   - projects
+  - title
 
 properties:
   $ref: "_definitions.yaml#/ubiquitous_properties"
 
-  contributor:
-    description: >
-      An entity responsible for making contributions to the resource. Examples of a Contributor include a person, an organization, or a service. Typically, the name of a Contributor should be used to indicate the entity.
-    type: string
+  # contributor:
+  #   description: >
+  #     An entity responsible for making contributions to the resource. Examples of a Contributor include a person, an organization, or a service. Typically, the name of a Contributor should be used to indicate the entity.
+  #   type: string
```

Comment:
Thanks for pointing this out Pauline. Yeah, these were intentially commented out after discussion with Phil about unused fields in our planned moved to a data package oriented framework (rather than individual files). However, I couldn't find where the configuration is located for the file page.. @paulineribeyre could you point me to the config for the file page?

I also think I am going to uncomment a few of these but it would be good to reduce a few of these.

Reply:
the file page is not configurable at the moment, it always uses the fields above